core/jazz_ui/src/environments/environment.oss.ts

```diff
@@ -6,6 +6,7 @@ export const environment = {
   envName: "oss",
   multi_env: {multi_env},
   slack_support: {slack_support},
+  WebLists: { "html" : "Html", "angular" : "Angular", "react": "ReactJS"},
```

Comment:
First option should be "static".  Also lets maintain camelcase for the key "webLists".